run_test_bandwidth.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#sh run_test_bandwidth.sh 2 2 0 10.20.1.81 22 
+set -e
```

Comment:
Can you provide a command example for launching this script for single-node multi-gpu and multi-node multi-GPU testing in README?

Reply:
Okay, sure. 
Just use the command in the second node GPU, change the $NODE_RANK parameter to 1, like:
sh run_test_bandwidth.sh 2 2 1 10.20.1.81 22 
And $NNODES represents the number of nodes.